test/utils.py

```diff
@@ -0,0 +1,52 @@
+"""Utils for tests"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+THIS_DIR = Path(os.path.dirname(__file__))
+PROJECT_ROOT = THIS_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+
+from backend.db.utils import get_db_connection, list_tables, run_sql
+
+
+class DbRefreshTestWrapper(unittest.TestCase):
+    """Runs common setup for all DB refresh related tests"""
+
+    @staticmethod
+    def populateMissingTestTables():
+        """Populate missing tables in test schema"""
+        n_rows = 50
+
+        # Detect
+        tables_by_schema = {
+            'n3c': [],
+            'test_n3c': [],
+        }
+        for schema in tables_by_schema:
+            con = get_db_connection(schema=schema)
+            tables_by_schema[schema] = list_tables(con, schema, filter_temp_refresh_tables=True)
+        missing_test_tables = set(tables_by_schema['n3c']) - set(tables_by_schema['test_n3c'])
+        # Populate
+        con = get_db_connection(schema='')
+        run_sql(con, f"""
+            DO $$ 
+            DECLARE
+                table_name text;
+            BEGIN
+                FOREACH table_name IN ARRAY {tuple(missing_test_tables)}
```

Comment:
## Populate missing tables in `test_n3c`
@Sigfried This will be helpful for several of current and upcoming tests. Sometimes we add a new table, and it is something that is needed in order for one of the DB refresh related tests to work. I've created a function that will run some SQL which will add any missing tables into `test_n3c`, and will populate with the first 50 rows from the same table in `n3c`.

There are a couple different test cases that can use this, so I've subclassed those test cases so that whenever they run, it detects automatically if anything is missing in `test_n3c` and adds.